apps/meteor/tests/data/livechat/rooms.ts

```diff
@@ -94,6 +94,22 @@ export const createDepartment = (agents?: { agentId: string }[]): Promise<ILivec
 	});
 }
 
+export const deleteDepartment = (departmentId: string): Promise<unknown> => {
+	return new Promise((resolve, reject) => {
+		request
+			.delete(api(`livechat/department/${departmentId}`))
+				.set(credentials)
+				.send()
+				.expect(200)
+				.end((err: Error, res: DummyResponse<ILivechatAgent>) => {
+					if (err) {
+						return reject(err);
+					}
+					resolve(res.body.user);
```

Comment:
```suggestion
					resolve();
```
The delete endpoint doesn't return this param.